### DISCUSSION TITLE
Need help with a few problems

### DISCUSSION THREAD

**Discussion by nookstray** [dialogic-godot/dialogic#843] [Q&A] [answered]
Hello! I’ve run into a few issues while using Dialogic and was hoping I could get them resolved here.

First, the choice button background textures and text color seem to reset every time I open Godot. Does anyone know what is causing this?

Second, is there any way to disable the choice hotkeys? My game only uses the arrow keys to make choices and not the mouse cursor, but since the hotkeys need to be set to something by default (I think) I can’t simply disable them. I’ve gotten around this issue by creating an action with no actual set input and setting the hotkeys to that. I was just wondering if there was a more efficient way to get around that.

Third, like I mentioned before, my game only uses the arrow keys and not the mouse cursor, and I was wondering if there was a way to set the choice hover sound effects to work with arrow keys instead of just the cursor.

Finally, I wanted to know what the correct way to end a dialogue and change scenes was. Do you need to end the dialog first and then change scene? Or vice versa? Do you omit the end dialog event entirely? Every time I change scenes, the new scene freezes. I would really appreciate any help with this.

**Comment by nookstray** [comment]
Oops, sorry! I think I might’ve put this in the wrong place. These types of questions go in the “Issues” tab, correct? If so, you can delete this. I’m new to GitHub so I’m not sure how to do it myself.

**Reply by Jowan-Spooner** [reply]
It's okay to ask here too. Issues is mostly for real or potentiall problems. Some of the things you described probably are, some are just questions.

**Comment by Jowan-Spooner** [comment, marked as answer]
Alright, I cannot answer everything, but some things.

A) I have never experienced theme settings resetting. If this keeps happening we need to create an issue and test this more.

B) I don't think there are any hotkeys by default, but I could me mistaken. The lack of a reset functionality has already been in #830.

C) There is no way to make the arrows make the choice sound in the exposed settings. However it would probably not be too complicated to implement yourself, if it's worth it for you.

D) There is no ultimate right way to change the scene, it depends what you want to achieve. In your case it seems like you pause the scene tree when starting the dialog. In that case you should somehow unpause it before changing the scene. Did you use emilios NPC tutorial?

Hope this helped. Feel free to ask further questions if something was unlcear.

**Reply by nookstray** [reply]
Thank you so much! I'll be sure to create an issue about the first problem now.
